eng/Versions.props

```diff
@@ -20,23 +20,35 @@
     <UsingToolVSSDK>true</UsingToolVSSDK>
```

Comment:
GitHub did a poor job at showing the diff. I suggest using "split view" to understand how and which lines were moved.